cmd/sqlflow/main_test.go

```diff
@@ -41,7 +42,9 @@ USING my_model;
 func TestWorkflow(t *testing.T) {
 	a := assert.New(t)
 	datasource := "mysql://root:root@tcp(127.0.0.1:3306)/?maxAllowedPacket=0"
-	code, e := compile("couler", testSQLProgram, datasource)
+	codegen := "couler"
+	log := log.WithFields(log.Fields{"codegen": codegen})
```

Comment:
1. Better to use the same scheme when calling the `log` package, use package function or use the object. Mixing two usages is a little bit confusing.
1. The object name should be `logger` not `log`.

suggest:

1. logging with no fields:
```go
logger := log.GetDefaultLogger()
logger.Info(...)
// or construct a default global logger variable
```
1. logging with fields

```sql
logger := log.WithFields(...)
logger.Info(...)
```

Reply:
Thanks for the suggestion which makes the log easier.